textdb/textdb-dataflow/src/main/java/com/google/re2j/PublicParser.java

```diff
@@ -0,0 +1,23 @@
+package com.google.re2j;
+
+/**
+ * Wrapper class for re2j.Parser
```

Comment:
We need high-level comments on why we do these "public" classes.  Write the trick we decided earlier.